modules/atomic-widgets/widgets/atomic-image.php

```diff
@@ -27,11 +28,13 @@ public function get_name() {
 	protected function render() {
 		$settings = $this->get_atomic_settings();
 
-		// TODO: Replace with actual URL prop
-		$image_url = $settings['url'];
+		$image = $settings['image'];
 
-		// phpcs:ignore WordPress.Security.EscapeOutput.OutputNotEscaped
-		echo "<img src='" . esc_url( $image_url ) . "' />";
+		?> <img
+			src='<?php echo esc_attr( $image['url'] ); ?>'
```

Comment:
```suggestion
			src='<?php echo esc_url( $image['url'] ); ?>'
```

(I would also just use the URL key in the variable `$image_url = $settings['image']['url']` 🤷 )